<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item alignment="Qt::AlignTop">
     <widget class="QFrame" name="rightFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="buttonFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="loadPushButton">
            <property name="text">
             <string>Загрузить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="savePushButton">
            <property name="text">
             <string>Сохранить</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="moveGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Перемещение</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="2">
           <widget class="QLabel" name="dyLabel">
            <property name="text">
             <string>dy = </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="dLineEdit">
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="dxLabel">
            <property name="text">
             <string>dx = </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="dxLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="dzLabel">
            <property name="text">
             <string>dz = </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLineEdit" name="dzLineEdit">
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QPushButton" name="movePushButton">
            <property name="text">
             <string>Переместить</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="scalingGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Масшабирование</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="kxLabel">
            <property name="text">
             <string>kx = </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QLineEdit" name="kzLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLineEdit" name="kyLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="kyLabel">
            <property name="text">
             <string>ky = </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="kxLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="kzLabel">
            <property name="text">
             <string>kz = </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QPushButton" name="scalingPushButton">
            <property name="text">
             <string>Масштабировать</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="rotationGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Поворот</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QLineEdit" name="fixLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="fizLabel">
            <property name="text">
             <string>Угол(0z) =</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="fizLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="fixLabel">
            <property name="text">
             <string>Угол(0x) =</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="fiyLabel">
            <property name="text">
             <string>Угол(0y) =</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLineEdit" name="fiyLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="rotationPushButton">
            <property name="text">
             <string>Повернуть</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cleanPushButton">
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
